src/server/tiering/small_bins.h

```diff
@@ -0,0 +1,60 @@
+// Copyright 2024, DragonflyDB authors.  All rights reserved.
+// See LICENSE for licensing terms.
+//
+
+#pragma once
+
+#include <absl/container/flat_hash_map.h>
+
+#include <optional>
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+#include "server/tiering/disk_storage.h"
+
+namespace dfly::tiering {
+
+// Small bins accumulate small values into larger bins that fill up 4kb pages.
+// SIMPLEST VERSION for now.
+class SmallBins {
+ public:
+  using FilledBin = std::pair<unsigned /* id */, std::string>;
```

Comment:
here actually add a comment that string is a serialized blob, that is at most 4KB but can be less.

Reply:
It's stated above in SmallBins that it fill 4kb pages